_includes/navbar.html

```diff
@@ -21,13 +21,11 @@
                 <li class="drop-down"><a href="/pages/documentation">Documentation</a>
                     <ul>
                         <li><a href="/pages/documentation/index.html#user-documentation">User documentation</a></li>
-                        <li><a href="/pages/documentation/index.html#developer-documentation">Developer documentation</a></li>
                         <li><a href="/pages/documentation/index.html#grid-model">Grid model</a></li>
                         <li><a href="/pages/documentation/index.html#grid-formats">Grid formats</a></li>
```

Comment:
```suggestion
                        <li><a href="/pages/documentation/index.html#grid-formats">Grid exchange formats</a></li>
```
to be consistent with the section title on the page ?

Reply:
ok